test/eth/Web3Service.spec.js

```diff
@@ -97,11 +97,13 @@ test('fetch version info on connect', done => {
     .manager()
     .connect()
     .then(() => {
-      expect(web3.version().api).toMatch(/^([0-9]+\.)*[0-9]+$/);
-      expect(web3.version().node).toMatch(
+      expect(web3.version().api).toMatch(
+        /^([0-9]+\.)*[0-9]([-][b][e][t][a][.]([0-9])*)*$/
+      );
+      expect(web3.version().node.toString()).toMatch(
         /^(Parity)|(MetaMask)|(EthereumJS.*)$/
       );
-      expect(web3.version().network).toMatch(/^[0-9]+$/);
+      expect(web3.version().network.toString()).toMatch(/^[0-9]+$/);
```

Comment:
`.toMatch` will only accept string arguments. web3-0.20.6 would return the network id as a string whereas the new version would return as a number.